src/WebJobs.Script/Description/DotNet/DotNetMuxer.cs

```diff
@@ -63,9 +63,9 @@ private static string TryFindMuxerPath()
             }
 
             var muxerDir = new FileInfo(fxDepsFile) // Microsoft.NETCore.App.deps.json
-                .Directory? // (version)
-                .Parent? // Microsoft.NETCore.App
-                .Parent? // shared
+                .Directory?// (version)
```

Comment:
nit: Don't like the recommendation in this instance... let's add a global suppression instead